docs/reference-guide/13-file-format.md

```diff
@@ -1,6 +1,31 @@
 # Study format changes
 This is a list of all recent changes that came with new Antares Simulator features. The main goal of this document is to lower the costs of changing existing interfaces, both GUI and scripts.
 
+## v8.6.0
+### Input
+A few changes related to the scenarization of RHS for binding constraints.
+
+- For each binding constraint, file **input/bindingconstraints/&lt;id&gt;.txt** is split into 3 files:
+    - **input/bindingconstraints/&lt;id&gt;_lt.txt**
+    - **input/bindingconstraints/&lt;id&gt;_gt.txt**
+    - **input/bindingconstraints/&lt;id&gt;_eq.txt**
```

Comment:
Why lt/gt => for range constraints
Also need to keep files when user change the type of constraint.
@Anterest team: careful not to delete/overwrite old file when user select a new type.